Updating .NET Overview file (#1131)
Samsung/tizen-docs: docs/application/dotnet/get-started/overview.md

# Get Started with .NET Applications

> When you create your first Tizen .NET application, you can select from the following options that use C# as the programming language:

Reviewer: ```suggestion
When you create your first Tizen .NET application, you can build your applications using C# language as follows:
```

Author: Based on your suggestion in https://github.com/Samsung/tizen-docs/pull/1130#discussion_r444121718, and to maintain consistency across all the get started pages, can we rephrase as:

You can create Tizen .NET applications in C#. You can also use the Tizen .NET APIs to access the Tizen platform-specific features. To create your first Tizen .NET application, you can select from the following profiles:

Please verify and share your valuable feedback.

> To create your first Tizen .NET application, you can select from the following options:

Author: Sure, I will make this change.

-   [Wearable .NET application](wearable/first-app.md)

>     Applications targeted to run on smartwatches. It can be optimized for small screens.

Reviewer 2: In this page, we have a list which lists up URLs to each "Create your first app" and descriptions for the URLs.
With this PR, the singular subjects in the descriptions are changed to plural.

However I'm not sure using plural here, while there is only one example in each "Create your first app" pages.

For example in line 5, 
\[Wearable .NET application](wearable/first-app.md) page is creating "an application", and the description sentence for this page is calling it as "Applications".

Another confusing point is that, these "Applications" are called as "It", not "They", in the following sentence.
> Applications ... It can be optimized for small screens.

How about checking the plural/singular used in the sentences again? In my thought singular form is required for each descriptions. Would you please let me know your thought?

Author: Thank you for pointing this.
I will use singular form.

Author: @mijin85cho 
This can be changed to:
An application targeted to run on smartwatches. It can be optimized for small screens.
or 
We can combine and rephrase as:
An application targeted to run on smartwatches and can be optimized for small screens.

Please verify and let me know your opinion on this.

Author: @mijin85cho 
Thank you for the suggestion.
Actually, the usage of "target" in both cases is fine. It is the difference in writing style. I can take your suggestion. However, "An applications" is the wrong usage. Therefore, to make it clear, I will rephrase as:

An application that is targeted to run on smartwatches. It can be optimized for small screens.
Please verify.

Reviewer 2: @Reni-90 Thanks for fixing my typo in the suggestion. Also I think your suggestion is acceptable to apply.

I hope you apply the update to this style for other description lines, 
for the sentences in overview for dotnet, web and native also,
to get the consistency between the pages.

Author: @mijin85cho 
Thank you very much for your feedback. 
I will use the same style for all other pages to maintain consistency.

>     Applications targeted to run on smartwatches. It can be optimized for small screens.

Author: I have tried to align with the original sentence to maintain minimal rephrases. 
Unifying sentences is a good suggestion. I will change it to:

Native: Tizen Native applications are created using C. You can use the available Tizen Native APIs to have advanced system settings and additional functionalities.
To create your first Tizen Native application, the following profiles are available:

.Net: Tizen .NET applications are created using C#.  You can use the available Tizen .NET APIs to have advanced system settings and additional functionalities.
To create your first Tizen .NET application, the following profiles are available:

Web: Tizen Web applications are created using the web-native languages, such as HTML, JavaScript, and CSS that are combined in a package.  You can use the available Tizen Web APIs to have advanced system settings and additional functionalities.
To create your first Tizen Web application, the following profiles are available:

Please verify.

>     Applications targeted to run on smartwatches. It can be optimized for small screens.

Reviewer 2: I trimmed your suggestion a little bit.

**Native:** Tizen Native applications are created using C. You can use Tizen Native API to have advanced system settings and additional functionalities.
To create your first Tizen Native application, following profiles are available:

**.Net:** Tizen .NET applications are created using C#. You can use Tizen .NET API to have advanced system settings and additional functionalities.
To create your first Tizen .NET application, following profiles are available:

**Web:** Tizen Web applications are created using the web-native languages, such as HTML, JavaScript, and CSS. You can use Tizen Web API to have advanced system settings and additional functionalities.
To create your first Tizen Web application, following profiles are available:

Indeed, I think even shorter sentence is possible as like below:

**Native:** Tizen Native applications are created using C. When you create your first Tizen Native application, you can select from the following options:

As you can see, most of my suggested sentence is re-using the original one.
Would you please check my suggestion, and share your opinion about it?
I hope to know the advantage of your suggestion that using the sentences in new style.

>     Applications targeted to run on smartwatches. It can be optimized for small screens.

Author: As we have Tizen native APIs for advanced system settings and additional functionalities, I wanted to keep that part in native applications. And you have suggested in your previous comment, not to focus on shortening the sentences. Therefore, I tried to unify all the sentences.
However, I too recommend keeping more crisp sentences.
So we can rephrase as:
Tizen Native applications are created using C. When you create your first Tizen Native application, you can select from the following options:

Based on this, I will unify the sentences throughout the page and in other PRs also.

-   [Wearable XAML application](wearable/first-xaml-app.md)

    Applications that allows you to use XAML to define the visual contents of a page and work together with a C# code-behind file on a smartwatch.

-   [TV .NET application](tv/first-app.md)

>     Applications targeted to run on smart TVs. It can be optimized for large screens and remote controls.

Reviewer 2: Suggestion.
I know your update in this PR is to shorten the sentences for the future work.
However, I'd like to suggest to focus on unifying the sentences and fixing the wrong singular/plural forms in the original sentences in this PR.

How about leaving the sentence shortening so that it can be checked together with the actual images when applying the UI?

Author: Sure, I will unify the sentences.
